backend-rust/CHANGELOG.md

```diff
@@ -12,6 +14,9 @@ All notable changes to this project will be documented in this file.
 - Add REST API `rest/balance-statistics/latest` for querying total amount in the network, used by external parties.
 - Add `ccdscan-api` option `--statement-timeout-secs` (env `CCDSCAN_API_DATABASE_STATEMENT_TIMEOUT_SECS`) for configuring a statement timeout the database connections and abort any statement that takes more than the specified amount of time. Defaults to 30 seconds.
 
+### Fixed
+
+- Alternate leading index on reward metrics
```

Comment:
Since there exist `alternate index` (secondary index), it is confusing.

```suggestion
- Reordered the primary key on the `reward_metrics` table from (block_slot_time, account_index) to (account_index, block_slot_time) to improve query performance.
```